pkg/minibroker/minibroker.go

```diff
@@ -510,14 +510,28 @@ func (c *Client) Bind(instanceID, serviceID, bindingID string, acceptsIncomplete
 	if acceptsIncomplete {
 		klog.V(3).Infof("minibroker: initializing asynchronous binding %q", bindingID)
 		go func() {
-			_ = c.bindSynchronously(instanceID, serviceID, bindingID, releaseNamespace, bindParams, provisionParams)
+			_ = c.bindSynchronously(
+				instanceID,
+				serviceID,
+				bindingID,
+				releaseNamespace,
+				BindParams(bindParams),
```

Comment:
Why is `bindParams` not already of type `BindParams`? That is, why can't `Broker.Bind()` do the wrapping?

Reply:
For some reason, I thought it implemented the upstream interface.

Reply:
Fixed with 1cb2a49.

Reply:
I thought so too! But then I checked, because I _know_ my memory is terrible :D